src/logging/default-logger.ts

```diff
@@ -50,6 +52,15 @@ export class DefaultLogger implements Logger {
     this.#rootPackageDir = rootPackage;
     this.#diagnosticPrinter = new DiagnosticPrinter(this.#rootPackageDir);
     this.console = ourConsole;
+    this.outputWrite = process.env['WIREIT_LOGGER_PREFIX'] ? this.outputWriteDebug : this.outputWriteDefault;
```

Comment:
Let's put this logic on our `Console` class as a new method that takes an output event and writes it out to the proper underlying stream, using the presence / absence of the env variable to determine whether to use the prefix. That way the implementation lives in a single place.